packages/kernels/upstream/definition.yaml

```diff
@@ -0,0 +1,6 @@
+category: "kernels"
+name: "linux"
+version: "6.5.7"
+labels:
```

Comment:
to get autobumps see: https://github.com/mocaccinoOS/desktop/blob/eeeaca01dd36143d82f166d481aa3462791e1ee2/packages/kernels/kernels/mocaccino/kernel/definition.yaml#L21 :taco: